.github/workflows/require-percy-tests.yml

```diff
@@ -0,0 +1,48 @@
+name: Require Percy specs
+
+on:
+  push:
+    branches:
+      - "!master"
+      - "!release-**"
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
```

Comment:
Isn't this going to create two triggers?
`pull_request` alone should be enough, unless I'm missing something.